mautic/mautic
# Add manually unsubsribed to segment filters

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | Y
| BC breaks? | N
| Deprecations? | N
| Issues | fixes https://github.com/mautic/mautic/issues/2782

[//]: # ( Required: )
#### Description:
It is requested to enable users to filter for *manually added DNC contacts* for **email** and **sms**.
Fields labeled:
 * Manually marked as do not contact - Email
 * Manually marked as do not contact - SMS

#### Steps to test this PR:
1. Create few segments containing current DNC reasons and their combination
2. Write down the numbers so you can verify
3. Checkout this PR and CC
4 . Verify that filters you have created in **1** are still valid.
5. Create segment containing new DNC fields and verify the content is valid.
